railties/lib/minitest/rails_plugin.rb

```diff
@@ -5,6 +5,17 @@
 require "rails/test_unit/runner"
 
 module Minitest
+  class BacktraceFilterWithFallback
+    def initialize(preferred, fallback)
+      @preferred = preferred
+      @fallback = fallback
+    end
+
+    def filter(backtrace)
+      @preferred.filter(backtrace).presence || @fallback.filter(backtrace)
```

Comment:
This depends on Active Support and I'm not seeing it being required. In reality I think we should not use `presence` here.